android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
\ No newline at end of file
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
I don't see the whole `android/gradle/wrapper` directory in `react-native-webview` or `react-native-permissions` (as an example for Android libraries) so I think we can go ahead and get rid of this and the other files in here.

Reply:
Does this mean we are removing `gradle wrapper` altogether? If so, should we also remove `gradlew` and `gradlew.bat`?

Reply:
Yup. I don't see why it would be needed for a library.